diff --git a/MLNT_cifar/config.py b/MLNT_cifar/config.py
index cf371ff..2da6131 100644
--- a/MLNT_cifar/config.py
+++ b/MLNT_cifar/config.py
@@ -52,10 +52,10 @@ crop = 4
 
 # -------------------------------------------- #
 
-#drive_dir = '.'
-#data_dir = './data/'
-drive_dir = '/content/drive/My Drive/Colab_Notebooks/git/PFM_Noisy_Labels/MLNT_cifar'
-data_dir =  '/content/drive/My Drive/Colab_Notebooks/git/PFM_Noisy_Labels/MLNT_cifar/data/'#'/data/'
+drive_dir = '.'
+data_dir = './data/'
+# drive_dir = '/content/drive/My Drive/Colab_Notebooks/git/PFM_Noisy_Labels/MLNT_cifar'
+# data_dir =  '/content/drive/My Drive/Colab_Notebooks/git/PFM_Noisy_Labels/MLNT_cifar/data/'#'/data/'
 
 #----------------------------------------------#
 #Dataloader files
